cluster/v1alpha1/types.go

```diff
@@ -12,47 +12,28 @@ import (
 
 // ManagedClusterSet defines a group of ManagedClusters that user's workload can run on.
 // A workload can be defined to deployed on a ManagedClusterSet, which mean:
```

Comment:
I'd prefer to have more specific language here that refers to other parts of the API. Not blocking feedback for this PR.

Reply:
I'm also asking myself how much of this content is really about subscription. Again, not blocking feedback.